src/plugins/displayLayout/components/TelemetryView.vue

```diff
@@ -169,11 +170,12 @@ export default {
                 return;
             }
 
+            let value;
             if (this.item.format) {
-                return printj.sprintf(this.item.format, this.datum[this.valueMetadata.key]);
+                value = this.formatter.format(this.datum, this.valueMetadata);
             }
 
-            return this.valueFormatter && this.valueFormatter.format(this.datum);
+            return value || this.valueFormatter && this.valueFormatter.format(this.datum);
```

Comment:
Remember our [code style rules around ternaries](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md#code-guidelines). This statement involves nested logic, so it's probably a good candidate for an `if` statement.